### DISCUSSION TITLE
encountered errors while performing template validation

### DISCUSSION THREAD

**Discussion by scyths** [projectdiscovery/nuclei-templates#9651] [General]
I encountered some problems while doing template validation，please help me, thanks!!!

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.2.5

		projectdiscovery.io

Started metrics server at localhost:9092
Could not find template 'technologies/sap-nw-abap-icm.yaml': could not find file: stat /root/nuclei-templates/technologies/sap-nw-abap-icm.yaml: no such file or directory
Could not find template 'technologies/sap-nw-abap-webgui.yaml': could not find file: stat /root/nuclei-templates/technologies/sap-nw-abap-webgui.yaml: no such file or directory
Could not find template 'vulnerabilities/sap-nw-abap-info-leakyaml.yaml': could not find file: stat /root/nuclei-templates/vulnerabilities/sap-nw-abap-info-leakyaml.yaml: no such file or directory
Could not find template 'yongyou-nc-beanshell-detect.yaml': could not find file: stat /root/nuclei-templates/yongyou-nc-beanshell-detect.yaml: no such file or directory
Could not find template 'yongyou-nc-beanshell-rce.yaml': could not find file: stat /root/nuclei-templates/yongyou-nc-beanshell-rce.yaml: no such file or directory
Could not find template 'network/saprouter-detect.yaml': could not find file: stat /root/nuclei-templates/network/saprouter-detect.yaml: no such file or directory
Could not find template 'vulnerabilities/saprouter-information-leak.yaml': could not find file: stat /root/nuclei-templates/vulnerabilities/saprouter-information-leak.yaml: no such file or directory
Could not find template 'CORS-initial-detect.yaml': could not find file: stat /root/nuclei-templates/CORS-initial-detect.yaml: no such file or directory
Could not find template 'basic-cors-misconfig.yaml': could not find file: stat /root/nuclei-templates/basic-cors-misconfig.yaml: no such file or directory
Could not find template 'CORS-null-origin-ACAC.yaml': could not find file: stat /root/nuclei-templates/CORS-null-origin-ACAC.yaml: no such file or directory
Could not find template 'CORS-valid-end-name.yaml': could not find file: stat /root/nuclei-templates/CORS-valid-end-name.yaml: no such file or directory
Could not find template 'CORS-arbitrary-domain-ACAC.yaml': could not find file: stat /root/nuclei-templates/CORS-arbitrary-domain-ACAC.yaml: no such file or directory
Could not find template 'CORS-arbitrary-subdomain-ACAC.yaml': could not find file: stat /root/nuclei-templates/CORS-arbitrary-subdomain-ACAC.yaml: no such file or directory
Could not find template 'CORS-period-regex-misconfig.yaml': could not find file: stat /root/nuclei-templates/CORS-period-regex-misconfig.yaml: no such file or directory
Could not find template 'thelabda/403_bypass_workflow/403-finder.yaml': could not find file: stat /root/nuclei-templates/thelabda/403_bypass_workflow/403-finder.yaml: no such file or directory
Could not find template 'thelabda/403_bypass_workflow/403-bypass_xheaders.yaml': could not find file: stat /root/nuclei-templates/thelabda/403_bypass_workflow/403-bypass_xheaders.yaml: no such file or directory
Could not find template 'thelabda/403_bypass_workflow/403-bypass_method.yaml': could not find file: stat /root/nuclei-templates/thelabda/403_bypass_workflow/403-bypass_method.yaml: no such file or directory
Could not find template 'thelabda/403_bypass_workflow/403-bypass_xrewriteurl.yaml': could not find file: stat /root/nuclei-templates/thelabda/403_bypass_workflow/403-bypass_xrewriteurl.yaml: no such file or directory
Could not find template 'thelabda/403_bypass_workflow/403-bypass_slash.yaml': could not find file: stat /root/nuclei-templates/thelabda/403_bypass_workflow/403-bypass_slash.yaml: no such file or directory
Could not find template 'exposed-panels/fortinet-panel.yaml': could not find file: stat /root/nuclei-templates/exposed-panels/fortinet-panel.yaml: no such file or directory
Could not validate templates: encountered errors while performing template validation

**Comment by princechaddha** [comment]
Can you share the command you are using to validate the templates? Also, please confirm whether the template exists at the specified path. If you suspect an issue with your setup, you can run 'nuclei -reset' 

Note: Using 'reset' will delete all the templates in '~/nuclei-templates' and update them with the latest template release. If you have added any templates to that directory, make sure to copy them. Also, make a copy of the configs if you use any
